packages/portfolio-contract/tools/portfolio-actors.ts

```diff
@@ -355,10 +355,14 @@ export const planTransfer = async (
   amount: NatAmount,
   feeBrand: Brand<'nat'>,
   makeGasEstimator: MakeGasEstimator,
+  accountIdByChain: Record<string, `${string}:${string}:${string}`>,
```

Comment:
```suggestion
  accountIdByChain: Record<string, AccountId>,
```

and in fact the record key is not just any string. Unless there's something I'm not seeing, I suggest:

```suggestion
  accountIdByChain: StatusFor['portfolio']['accountIdByChain'],
```